src/main/java/micdoodle8/mods/galacticraft/core/entities/player/FreefallHandler.java

```diff
@@ -345,37 +345,32 @@ public void postVanillaMotion(EntityPlayerSP p) {
                 doCentrifugal = spinManager.updatePlayerForSpin(p, 1F);
             }
 
-            // Do freefall motion
-            if (!p.capabilities.isCreativeMode) {
-                FreefallHandler.freefallMotion(p);
```

Comment:
Since it's no longer used this function can be removed.

Reply:
@27yearoldminecraftgaymer why did you mark this as resolved? You didn't remove the function `freefallMotion`

Reply:
commented it out